lms/djangoapps/api_manager/users/views.py

```diff
@@ -36,9 +38,11 @@
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import UsageKey, CourseKey
 from opaque_keys.edx.locations import Location, SlashSeparatedCourseKey
+from openedx.core.djangoapps.user_api.preferences.api import set_user_preference
 from student.models import CourseEnrollment, PasswordHistory, UserProfile
 from openedx.core.djangoapps.user_api.models import UserPreference
 from student.roles import CourseAccessRole, CourseInstructorRole, CourseObserverRole, CourseStaffRole, CourseAssistantRole, UserBasedRole
+from openedx.core.djangoapps.user_api.models import UserPreference
```

Comment:
Why are we importing UserPreference but removing the reference to it on line 343?